helmfile/charts/networkpolicy-service/templates/coredns/allow-coredns.yaml

```diff
@@ -0,0 +1,49 @@
+{{ if .Values.coredns.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-coredns
+spec:
+  podSelector:
+    matchLabels:
+      k8s-app: kube-dns
+  policyTypes:
+    - Egress
+    - Ingress
+  ingress:
+    - ports:
+      - port: 53
+        protocol: UDP
+      - port: 53
+        protocol: TCP
+    - from:
+        - podSelector:
+            matchLabels:
+              app: prometheus
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: monitoring
+      ports:
+        - port: 9153
+  egress:
+    - to:
+      {{- range $IP := .Values.global.scApiserver.ips }}
+      - ipBlock:
+            cidr: {{ $IP }}
+        {{- end }}
+      ports:
+        - port: {{ .Values.global.scApiserver.port }}
+    - to:
+      {{- range $IP := .Values.coredns.externalDns.ips }}
```

Comment:
Is the plan that we should config all the ips that this should be able to talk to? Or is the thinking that we're going to open the whole pod IP range?

Reply:
I like the idea of putting only the ip's that it talks too and not open for all ip's
Also I looked at some Safespring and Cleura env and these ip's were the same
Do not know about other clouds especially non-openstack ones though....